asdf-pyapp fails when pip version string not in major.minor.patch format

**Describe the bug**

asdf-pyapp fails to install an app with pip version 21.0

**Steps to reproduce**

$ python3 -m pip --version
pip 21.0 from /usr/lib/python3.10/site-packages/pip (python 3.10)

$ asdf install yamllint
asdf-pyapp: [ERROR] Unable to parse pip version

With shell trace:
```
++ local 'regex=pip (.+) from.*\(python (.+)\)'
++ [[ pip 21.0 from /usr/lib/python3.10/site-packages/pip (python 3.10) =~ pip (.+) from.*\(python (.+)\) ]]
++ echo -n 21.0
+ pip_version=21.0
+ [[ 21.0 =~ ^([0-9]+)\.([0-9]+)\. ]]
+ fail 'Unable to parse pip version'
+ echo -e 'asdf-pyapp: [ERROR] Unable to parse pip version'
asdf-pyapp: [ERROR] Unable to parse pip version
```

<!-- Steps to reproduce the behavior: -->

**Expected behavior**

asdf-pyapp working with PIP version specified in major.minor format.

<!-- A clear and concise description of what you expected to happen. -->

--- a/lib/utils.bash
+++ b/lib/utils.bash
@@ -128,7 +128,7 @@ get_package_versions() {
 
   local pip_version
   pip_version=$(get_python_pip_versions "$ASDF_PYAPP_RESOLVED_PYTHON_PATH")
-  if [[ $pip_version =~ ^([0-9]+)\.([0-9]+)\. ]]; then
+  if [[ $pip_version =~ ^([0-9]+)\.([0-9]+)\.? ]]; then
     local pip_version_major=${BASH_REMATCH[1]}
     local pip_version_minor=${BASH_REMATCH[2]}
   else

